apps/app/src/components/AdminPanel/AdminPanelQuestionsList.tsx

```diff
@@ -8,31 +8,36 @@ type AdminPanelQuestionsListProps = Readonly<{
 	refetchQuestions: () => void;
 }>;
 
-export const AdminPanelQuestionsList = ({
-	questions,
-	refetchQuestions,
-}: AdminPanelQuestionsListProps) => {
-	const serializedQuestions = questions.map((question) =>
-		use(serializeQuestionToMarkdown(question)),
-	);
+export const AdminPanelQuestionsList = memo(
+	({ questions, refetchQuestions }: AdminPanelQuestionsListProps) => {
+		const serializedQuestions = questions.map((question) =>
+			use(
+				(async () => {
+					const serializedQuestion = await serializeQuestionToMarkdown(question);
+					return { content: question.question, ...serializedQuestion };
```

Comment:
Tak nie będzie czytelniej? Bo teraz nie jest dla mnie jasne jakie własności zwraca serializeQuestionToMarkdown
```suggestion
					return { ...question, mdxContent: serializedQuestion.mdxContent };
```